specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/examples/ManagedInstanceCreateMax.json

```diff
@@ -48,7 +50,8 @@
           "subnetId" : "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Network/virtualNetworks/vnet1/subnets/subnet1",
           "state": "Ready",
           "collation": "SQL_Latin1_General_CP1_CI_AS",
-          "dnsZone": "1b4e2caff2530"
+          "dnsZone": "1b4e2caff2530",
```

Comment:
Is publicDataEndpointEnabled intentionally not there in the response?

Reply:
Yes, it's by design.

Reply:
I talked with Mladen who was implementing this feature. He actually forgot to include it here. It should be in response. I'll update it and he'll make sure to update his part.